ovirtlago/cmd.py

```diff
@@ -35,15 +35,6 @@
     workdir_class=ovirtlago.OvirtWorkdir,
 )
 # TODO: Remove this, and properly complain on unset config values
```

Comment:
nope, theres still
```
DISTS = ['el6', 'el7', 'fc20']
```
which I will remove in a next clean-up patch(there many other stuff here too..)